src/DataTable/DataTable.js

```diff
@@ -335,7 +335,6 @@ const DataTable = memo(({
                     className="rdt_TableHeadRow"
                     role="row"
                     dense={dense}
-                    disabled={progressPending || data.length === 0}
```

Comment:
Can you also remove the css check in https://github.com/jbetancur/react-data-table-component/blob/6e9d9246076f9b39663c942fc69848ec6744edde/src/DataTable/TableHeadRow.js#L13